template.pkr.hcl

```diff
@@ -42,47 +50,70 @@ source "amazon-ebs" "ubuntu" {
   }
   ssh_username = "ubuntu"
   tags = {
-    Name       = "Coder"
-    OS_Version = "Ubuntu"
+    name       = "Coder"
+    os         = "Ubuntu"
+  }
+}
+
+source "googlecompute" "ubuntu" {
+  project_id      = "coder-enterprise-market-public"
+  auth_f
```

Comment:
what does this do? seems like invalid HCL?

Reply:
it was a typo left while I was testing with `auth_file`. We do not need it anymore as we will be using google-work-identity for auth. Right?